### ISSUE TITLE
Error: No available tap caskroom/versions

### ISSUE THREAD

**Issue by yozlet** [18F/laptop#133]
Got this:

```
Homebrew already installed. Skipping ...
Uninstalling brew-cask... (2,374 files, 1.4M)
Error: No available tap caskroom/versions.
failed
```

OS X 10.12.3; have been on 10.12 for a little while. This is (I believe) my first time running the `laptop` script, and this has been my 18F work machine for over 2 years, so I've had Homebrew and lots of other things installed for ages.

Laptop.log:
```
Want to switch your shell from the default ESC[1mbashESC(BESC[m to ESC[1mzshESC(BESC[m?
Both work fine for development, and ESC[1mzshESC(BESC[m has some extra
features for customization and tab completion.
If you aren't sure or don't care, we recommend ESC[1mzshESC(BESC[m.
Note that you can always switch back to ESC[1mbashESC(BESC[m if you change your mind.
Please see the README for instructions.
Press ESC[1myESC(BESC[m to switch to zsh, ESC[1mnESC(BESC[m to keep bash:
=== Getting ready to change your shell to zsh. Please enter your password to continue. ===
=== Note that there won't be visual feedback when you type your password. Type it slowly and press return. ===
=== Press control-c to cancel ===
Changing shell for yoz.

Homebrew already installed. Skipping ...
Uninstalling brew-cask... (2,374 files, 1.4M)
Error: No available tap caskroom/versions.
failed
```

**Comment by monfresh** [comment]
Thanks for the report. I added the following check due to some changes a while back with brew-cask:
```
if brew_is_installed 'brew-cask'; then
  brew uninstall --force 'brew-cask'
  brew untap caskroom/versions
fi
```
Sounds like it's possible for `brew-cask` to be installed without the `caskroom/versions` tap. I will add a check to see if the tap is present before attempting to untap it.

**Comment by monfresh** [comment]
@yoz I fixed this in #137. Once it gets merged, I'll ping you to give it another shot.

**Comment by yozlet** [comment]
Yay, thank you!

On Wed, Jan 25, 2017 at 11:44 AM, Moncef Belyamani <notifications@github.com
